grpc/pb/space.proto

```diff
@@ -37,9 +37,8 @@ message ListDirectoriesResponse {
 }
 
 message ConfigInfoResponse {
-  string folderPath = 1;
-  string port = 2;
-  string appPath = 3;
+  string port = 1;
```

Comment:
need to coordinate with FE to sync on proto schema, since this could break the release on their side